packages/server-wallet/src/wallet/store.ts

```diff
@@ -290,6 +290,17 @@ export class Store {
     return (await Channel.query(this.knex)).map(channel => channel.protocolState);
   }
 
+  /**
+   * Returns all channels that are not finalized chain
+   */
+  async getActiveChannels(): Promise<ChannelState[]> {
```

Comment:
I would prefer we keep the terminology as small as possible, so suggest `getOpenChannels` (see https://github.com/statechannels/statechannels/pull/3134#discussion_r559580530)

Reply:
Unfortunately `open` may be overloaded in our `wallet` as we use open as a channel status as well. I've just updated this to `getNonFinalizedChannels` which is a bit clunky but clearer